### PR TITLE
Add optional db backup

### PR DESCRIPTION
**Pull request by Baggerone**

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -24,3 +24,23 @@ output "aws_ses_secret_access_key" {
   value       = aws_iam_access_key.attachments.secret
   sensitive   = true
 }
+
+output "bkup_key_id" {
```
Comment (Schparky) [inline outputs.tf:28]: Didn't occur to me before, but what is the purpose of these outputs?

File: vars.tf [hunk 2]
```diff
@@ -229,3 +229,19 @@ variable "user_welcome_email_ending" {
   type        = string
   default     = ""
 }
+
+variable "enable_db_backup" {
+  description = "Whether to have a database backup or not"
+  type    = bool
+  default = false
```
Comment (Schparky) [inline vars.tf:236]: Curious how you decided on `false` as the default.

Comment (Baggerone) [reply vars.tf:236]: Mostly to avoid backups on staging environments.

Comment (Schparky) [reply vars.tf:236]: Ah, good thought.

File: outputs.tf (hunk 1 above)
Comment (Baggerone) [reply outputs.tf:28]: I assumed we would need access to the key_id at least, but maybe we can just get everything from the AWS UI. Which ones would you like me to remove?

Comment (Schparky) [reply outputs.tf:28]: I generally think of outputs as data that's needed as an input to another module/workspace. Sometimes we use them as a convenience for not having to dig into the state. No need to remove any, but a comment might help to know why they exist.

**Review by Schparky — APPROVED** [review]

**Pull request merged**